src/angular-hybrid.ts

```diff
@@ -117,21 +117,26 @@ export class UIViewNgUpgrade {
 
     // Expose getters on PARENT_INJECT for context (creation state) and fqn (view address)
     // These will be used by further nested UIView
-    Object.defineProperty(parent, "context", {
-      get: function() {
-        const data = ng1elem['inheritedData']('$uiView');
-        return (data && data.$cfg) ? data.$cfg.viewDecl.$context : registry.root();
-      },
-      enumerable: true
-    });
+    if (typeof parent.context === 'undefined') {
```

Comment:
I've found the underlying issue here I believe. `@Inject(UIView.PARENT_INJECT)` is provided by
``` js
// line 99
viewProviders: [ { provide: UIView.PARENT_INJECT, useValue: { } } ],
```
If you have two upgraded ui-view components, the first one will receive the object in `useValue`, and define the `context` and `fqn` properties when it is constructed. When the second ui-view upgraded component is constructed, that same object from `useValue` is injected and `context` and `fqn` are already defined (by the previous component).

I believe each ui-view component should have it's own copy of `UIView.PARENT_INJECT`, so `useFactory` should be used instead of `useValue`.  @christopherthielen

Reply:
It's interesting that you got `TypeError: Cannot redefine property: context`. In my case, the properties were just overridden without error, leaving the first ui-view component empty. Very difficult to debug!